templates/signed-out.html

```diff
@@ -1,11 +1,34 @@
-{% extends 'layouts/_base.html' %}
+{% extends 'errors/_base.html' %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
 
 {% set page_title = _("Signed out") %}
 
 {% block main %}
-  <h1>{{ _("Your survey answers have been saved. You are now signed out") }}.</h1>
 
-  {% if account_service_log_out_url %}
-    <a href="{{account_service_log_out_url}}">{{ _("Return to your account") }}</a>
+  {{
+    onsPanel({
+      "id": '',
+      "type": 'success',
+      "iconType": 'check',
+      "iconSize": 'xl',
+      "classes": "ons-u-mb-xl",
+      "body": _("<h1>Your progress has been saved</h1>")
+    })
+  }}
+
+  {% if survey_type and (survey_type in SURVEY_TYPES_BUSINESS or survey_type in SURVEY_TYPES_DEFAULT) %}
```

Comment:
We can make it cleaner by doing:
```
  {%  if survey_type %}
    {% if survey_type in SURVEY_TYPES_BUSINESS + SURVEY_TYPES_DEFAULT and business_logout_url %}

        <p>{{ _("To find further information or resume the survey, <a href='{url}'>return to My Account</a>.").format(url = business_logout_url) }}</p>

    {% elif survey_type in SURVEY_TYPES_SOCIAL + SURVEY_TYPES_HEALTH and other_logout_url %}

        <p>{{ _("To resume the survey, <a href='{url}'>re-enter your access code</a>.").format(url = other_logout_url) }}</p>

    {% endif %}
  {%  endif %}
```

Reply:
The if survey type can be removed. You will always have a survey type for this page.